lib/model/message_list.dart

```diff
@@ -139,14 +156,25 @@ mixin _MessageSequence {
   /// It exists as an optimization, to memoize the work of parsing.
   final List<ZulipMessageContent> contents = [];
 
+  /// The messages sent by the self-user, retrieved from
+  /// [MessageStore.outboxMessages].
+  ///
+  /// See also [items].
+  ///
+  /// Usually this should not have that many items, so we do not anticipate
+  /// performance issues with unoptimized O(N) iterations through this list.
+  final List<OutboxMessage> outboxMessages = [];
```

Comment:
This adds some overhead in magnitude of O(1) (where the constant is
    the number of outbox messages in a view, expected to be small) on
    message event handling.

That is not what the word "constant" means :slightly_smiling_face: 

Instead can say:

> This adds some overhead on message-event handling, linear in the number of outbox messages in a view. We rely on that number being small.

Or could say something like "O(k), where there are k outbox messages in the view".